iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -173,7 +176,8 @@ internal sealed class MqttTransportHandler : TransportHandler, IMqttIotHubEventH
             _deviceboundMessageFilter = string.Format(CultureInfo.InvariantCulture, DeviceBoundMessagesTopicFilter, iotHubConnectionString.DeviceId);
             _deviceboundMessagePrefix = string.Format(CultureInfo.InvariantCulture, DeviceBoundMessagesTopicPrefix, iotHubConnectionString.DeviceId);
 
-            _qos = settings.PublishToServerQoS;
+            _qosSendPacketToService = settings.PublishToServerQoS;
```

Comment:
It looks like preview needs an update from main first, because I think these changes came from main through your feature branch.

@abhipsaMisra, can you assist Dylan here with getting the preview branch ready to merge into?

Reply:
I see, this PR is to bring main to preview.